src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -265,14 +270,14 @@ class TableView extends React.Component<TableViewProps> {
   }
 }
 
-const ExpandAggregateRow = (props: {
-  children: ({willExpand: boolean}) => React.ReactNode;
+function ExpandAggregateRow(props: {
```

Comment:
Just out of curiosity - do we prefer functions instead of arrow functions assigned to const?

Reply:
I try to only use arrow functions when I need the binding behavior.